README.md

```diff
@@ -220,7 +220,11 @@
 
   ```
   yum install ruby
+  yum install ruby-devel
+  yum install zlib-devel
+
   gem install trollop
+  gem install fog
```

Comment:
Yeah, I need to get back to that, I started down that route here  where I needed a project skeleton (gemfile/gemspec/rspec) #36, haven't finished that one up yet